modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/selectionFilters.spec.ts

```diff
@@ -18,8 +18,10 @@ const SELECTION_API_HEADLESS_FILTER_NAME = 'Selection API Headless filter';
 const SELECTION_PICKLIST_FILTER_NAME = 'Selection Picklist filter';
 const SELECTION_PICKLIST_NO_PRESELECTED_VALUES_FILTER_NAME =
 	'Selection Picklist filter without preselected values';
-const PICKLIST_VALUE_KEY = 'sampleValue';
-const PICKLIST_VALUE_NAME = 'Sample Value';
+const PICKLIST_VALUE_KEY_1 = 'sampleValue1';
+const PICKLIST_VALUE_NAME_1 = 'Sample Value 1';
+const PICKLIST_VALUE_KEY_2 = 'sampleValue2';
+const PICKLIST_VALUE_NAME_2 = 'Sample Value 2';
```

Comment:
This would work fine, but in general it's a good practice to use `getRandomString` for all sample labels. The main reason is that test doesn't fail in a messy environment, such as entries existing that you manually created locally, or previous tests that you stopped or cleanup failed.

Reply:
Yes, absolutely!